From a36cf24875bdd8d5eedc39ad835900232880987c Mon Sep 17 00:00:00 2001
From: Adhemerval Zanella <adhemerval.zanella@linaro.org>
Date: Wed, 10 Aug 2022 17:56:13 -0300
Subject: [PATCH 155/163] intl: Use strcpy on _nl_make_l10nflist

It avoid compiler to turn is on strcpy, which might generate a strcpy
PLT call since there is no explicit redirection.
---
 intl/l10nflist.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/intl/l10nflist.c b/intl/l10nflist.c
index 078a450dfe..1945d3f873 100644
--- a/intl/l10nflist.c
+++ b/intl/l10nflist.c
@@ -210,7 +210,7 @@ _nl_make_l10nflist (struct loaded_l10nfile **l10nfile_list,
     }
 
   *cp++ = '/';
-  stpcpy (cp, filename);
+  strcpy (cp, filename);
 
   /* Look in list of already loaded domains whether it is already
      available.  */
-- 
2.37.2

